jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -104,7 +104,7 @@ async def job_submission_create(
 
     logger.debug(f"Job-submission created: {job_submission_data=}")
 
-    return job_submission_data
+    return JobSubmissionResponse(**job_submission_data)  # type: ignore
```

Comment:
Why does this require an ignore annotation?

Reply:
we've been discussing this privately on: https://omnivector.slack.com/archives/C03FP13MAE6/p1669737235008909

Reply:
The issue is that the type of `job_submission_data` is not a dict. This can be fixed by using the `from_orm()` method that initializes the response object from another object:

```python
return JobSumbissionResponse.from_orm(job_submission_data)
```

Mypy accepts this just fine.

Reply:
Mypy accepts it, but pydantic does not, there is an issue related to this: https://github.com/tiangolo/pydantic-sqlalchemy/issues/19

I'm leaving the code as it is for now.